src/cogniteClient.ts

```diff
@@ -291,12 +291,52 @@ export default class CogniteClient {
   public getMetadata = (value: any) => this.metadataMap.get(value);
 
   /**
-   * Do plain get-requests using the client
+   * Basic HTTP method for GET
+   *
+   * @param path The URL path
```

Comment:
Maybe add an example. Unclear if `/api/v1` is automatically included in the request for example

Reply:
Made an example, please review